src/include/daos_srv/daos_server.h

```diff
@@ -258,10 +292,9 @@ struct dss_module_ops {
 	int	(*dms_profile_stop)(void);
 };
 
-int srv_profile_stop(struct srv_profile *sp);
+int srv_profile_stop();
```

Comment:
(style)  Bad function definition - int srv_profile_stop() should probably be int srv_profile_stop(void)